ZhuoHeiChaUI/Events/NotifyReturnTributeEventArgs.cs

```diff
@@ -7,5 +7,7 @@ namespace ZhuoHeiChaUI.Events
 {
     public class NotifyReturnTributeEventArgs : EventArgs
     {
+        public int payer;
+        public int cardsToBeReturnCount;
```

Comment:
Same here. Use properties and `PascalCase`.

Reply:
Done